IanGRap/Squish
# Volumetric Revision

Contributors: Ian

Goal: Change our volumetric lighting program to one that is prettier and has a more robust feature set (Its called Aura).

Important Files:
- Sprint 5.unity: The lights here have been replaced with volumetric lights. The spotlights should be the best example. They shouldn't have the weird artifacting the old ones did.
- Assets/Aura/Aura - Starting Guide: Outlines all the features in Aura. Does it seem interesting?

How to test:
Load up Sprint 5 scene and you should see the updated lights. Play through a full match in a build and hopefully there are no bugs. Take a look at the starting guide and see if this is something worth exploring.

Notes: IMPORTANT. Opening a file with Aura for the first time takes a while. YOUR COMPUTER IS NOT BROKEN. It might take about 20 minutes to compile the shader changes.
